### ISSUE TITLE
Set maximum multi-image frame iterations

### ISSUE THREAD

**Issue by mcbouslog** [zooniverse/Panoptes-Front-End#5432]
## Expected behavior
Infinite multi-image play iterations is not an accessible UX.
Frame iterations should be capped at __(?).

## Current behavior
If set in workflow:
![Screen Shot 2019-07-02 at 10 46 25 AM](https://user-images.githubusercontent.com/12118751/60544210-f2663200-9cdd-11e9-857a-977f48f9d9dc.png)

Then the following code infinitely loops through multi-image subject frames:
https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/components/subject-viewer.cjsx#L321

https://www.zooniverse.org/projects/marckuchner/backyard-worlds-planet-9/classify

**Comment by eatyourgreens** [comment]
The [WCAG guidance on moving information](https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-pause.html) suggests a pause/stop button should be available for animation that lasts more than 5s. We already have a pause button, so that's fine, but we could also consider setting the maximum playback duration to 5s.
